draft-irtf-cfrg-bbs-signatures.md

```diff
@@ -584,21 +727,63 @@ ABORT if:
 
 Procedure:
 
-1.  (Q_1, MsgGenerators) = create_generators(L+1, PK)
-2.  (H_1, ..., H_L) = MsgGenerators
-3.  (H_i1, ..., H_iR) = (MsgGenerators[i1], ..., MsgGenerators[iR])
-4.  (H_j1, ..., H_jU) = (MsgGenerators[j1], ..., MsgGenerators[jU])
-5.  domain = calculate_domain(PK, Q_1, (H_1, ..., H_L), header)
-6.  D = P1 + Q_1 * domain + H_i1 * msg_i1 + ... + H_iR * msg_iR
-7.  T =  Abar * r2^ + Bbar * r3^ + H_j1 * m^_j1 + ... +  H_jU * m^_jU
-8.  T = T + D * c
-9.  cv = calculate_challenge(Abar, Bbar, T, (i1, ..., iR),
-                             (msg_i1, ..., msg_iR), domain, ph)
-10. if c != cv, return INVALID
-11. if e(Abar, W) * e(Bbar, -BP2) != Identity_GT, return INVALID
-12. return VALID
+1. (Q_1, MsgGenerators) = create_generators(L+1, PK)
```

Comment:
Maybe we can rename MsgGenerators to msg_generators for consistency.

Reply:
Not sure. I like the list of "Points" starting with a capital letter TBH. Not feeling strongly about it!